Common/SimConfig/include/SimConfig/SimParams.h

```diff
@@ -41,6 +41,7 @@ struct SimCutParams : public o2::conf::ConfigurableParamHelper<SimCutParams> {
 // parameter influencing material manager
 struct SimMaterialParams : public o2::conf::ConfigurableParamHelper<SimMaterialParams> {
   float globalDensityFactor = 1.f;
+  std::string localDensityFactor;
```

Comment:
could you please add a comment to this data member describing the format expected to be given by the user? Also, we should clarify if localDensityFactor takes precedence over globalDensityFactor (or if they act in a multiplicative way).

Reply:
Hi Sandro, thank you very much for the review! I added the comments to help the user.